.github/workflows/release.yaml

```
@@ -0,0 +1,25 @@
+name: Release
+
+on:
+  push:
+    branches:
+      # the default branch
+      - master
+
+permissions:
+  contents: write
+
+jobs:
+  # This mirrors build_and_test workflow
+  build_and_test:
+    name: 'Build and Test'
+    uses: StackStorm-Exchange/ci/.github/workflows/pack-build_and_test.yaml@master
+    with:
+      enable-common-libs: true
+      #apt-cache-version: v0
+      #py-cache-version: v0
+
+  tag_release:
+    needs: build_and_test
```

Maybe add another `integration_test` workflow that converts the integration_test jobs from CircleCI to GHA.
Make that a reusable workflow (`on.workflow_call`), and then call it in both `build_and_test` and `release` workflows.

```suggestion
  integration_tests:
    name: Integration Tests
    uses: StackStorm-Exchange/stackstorm-zabbix/.github/workflows/integration_tests.yaml@master

  tag_release:
    needs:
      - build_and_test
      - integration_tests
```